### PR TITLE
Added an array of ghost element flags to mesh properties for parallel VTU ouput. 

### PR DESCRIPTION
**Pull request by wenqing**
As titled. Added the array with memory usage as smaller as possible.

### CODE REVIEW

File: MeshLib/IO/MPI_IO/NodePartitionedMeshReader.cpp [hunk 1]
```diff
@@ -95,6 +97,23 @@ MeshLib::NodePartitionedMesh* NodePartitionedMeshReader::read(
...
+            "vtkGhostType", MeshLib::MeshItemType::Cell, 1);
+    if (!vtkGhostType)
+    {
+        WARN(
+            "Could not create PropertyVector for the cell values of "
+            "vtkGhostType in Mesh.");
+    }
+    else
+    {
+        std::fill_n(std::back_inserter(*vtkGhostType),
```
Comment (endJunction) [inline MeshLib/IO/MPI_IO/NodePartitionedMeshReader.cpp]: Minor. You could `reserve(_mesh_info.regular_elements + _mesh_info.ghost_elements)` before filling the vector to superb performance ;)

**Comment by endJunction** [comment]
@rinkk Could you have a look on this and tell if it works for you?

**Comment by wenqing** [comment]
@rinkk  Would you please test it?

File: MeshLib/IO/MPI_IO/NodePartitionedMeshReader.cpp (hunk 1 above)
Comment (wenqing) [reply MeshLib/IO/MPI_IO/NodePartitionedMeshReader.cpp]: Added.

**Comment by bilke** [comment]
@wenqing I talked with Karsten: he cannot review it in the next days. As we want to have it in the release I will merge this now. Karsten will test it later on...

**Comment by wenqing** [comment]
@wenqing Thanks. It can output the required cell property in vtu file.

**Pull request merged**